### PR TITLE
Add Response

### PR DESCRIPTION
**Pull request by jpslav**

### CODE REVIEW

File: oxlate/cookie.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import re
+import datetime
+
+class Cookie:
+    def __init__(self, name, value=None, expires_at=None, expires_days_from_now=None, path=None, domain=None):
```
Comment (kevinburleigh75) [inline oxlate/cookie.py]: Add `expires_immediately=True` option?

File: oxlate/headers.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from .utils import string_to_case_number, case_number_to_string
+
+class Headers:
+    def __init__(self):
+        self._data = {}
+
+    def add(self, name, value, adjust_case_to_allow_duplicates=False):
```
Comment (kevinburleigh75) [inline oxlate/headers.py:11]: Add link to description of case problem?
    https://forums.aws.amazon.com/thread.jspa?messageID=701434

File: oxlate/headers.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        # import ipdb; ipdb.set_trace()
+        if adjust_case_to_allow_duplicates:
+            if self._data.get(name):
+                # Collision, need to change case
+                same_names_case_insensitive = list(filter(lambda key: key.lower() == name.lower(), self._data.keys()))
+                existing_names_case_binary_representations = set(map(string_to_case_number, same_names_case_insensitive))
```
Comment (kevinburleigh75) [inline oxlate/headers.py]: I found `case_number` to be a bit confusing - perhaps `case_indicator`?  Open to other suggestions...

File: oxlate/headers.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                available_case_binary_representations = sorted(set(range(pow(2,len(name)))) - existing_names_case_binary_representations)
```
Comment (kevinburleigh75) [inline oxlate/headers.py]: I think this formula might be incorrect.  2^N works if every character has upper- and lower-case options, but `-` (as in `set-cookie`) does not.  So really it should be 2^A where A is the number of alpha (a-z) characters.

Comment (kevinburleigh75) [reply oxlate/headers.py]: This code is also exponentially slow in `len(name)` - that could get ugly fast.

File: oxlate/response.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    def set_content_type_html(self):
+        self._content_type = "text/html"
+        return self
+
+    def set_body(self, body):
+        self._body = body
+        return self
+
+    def add_cookie(self, name, value=None, expires_at=None, path=None, domain=None):
```
Comment (kevinburleigh75) [inline oxlate/response.py:35]: As above, perhaps add some sort of `expires_immediately=True` option?

File: oxlate/cookie.py (hunk 1 above)
Comment (jpslav) [reply oxlate/cookie.py]: added `delete_cookie` to response

File: oxlate/headers.py (hunk 4 above)
Comment (jpslav) [reply oxlate/headers.py]: fixed

File: oxlate/response.py (hunk 5 above)
Comment (jpslav) [reply oxlate/response.py:35]: added `delete_cookie`

**Review by kevinburleigh75 — APPROVED** [review]

**Pull request merged**